ansible/roles/test/tasks/crm/crm_test_acl_counter.yml

```diff
@@ -0,0 +1,62 @@
+- block:
+
+    - set_fact: crm_stats_acl_counter_used=0
+    - set_fact: crm_stats_acl_counter_available=0
+
+    - name: Copy ACL JSON config to switch.
+      copy: src=roles/test/tasks/crm/acl.json dest=/tmp
+
+    - name: Add ACL
+      command: acl-loader update full /tmp/acl.json
```

Comment:
Just to confirm, @lguohan, is `DATAACL ` table always present in the DUTs or is there any case where it may be missing. If so, the acl-loader will fail since it adds the entry to `DATAACL`